src/content/docs/en/guides/deploy/github.mdx

```diff
@@ -8,82 +8,13 @@ i18nReady: true
-
-## Configure Astro for GitHub Pages
-
-### Deploying to a `github.io` URL 
-
-Set the [`site`](/en/reference/configuration-reference/#site) and [`base`](/en/reference/configuration-reference/#base) options in `astro.config.mjs`.
-    
-```js title="astro.config.mjs" ins={4-5}
-import { defineConfig } from 'astro/config'
-
-export default defineConfig({
-  site: 'https://astronaut.github.io',
-  base: 'my-repo',
-})
-```
-
-#### `site`
-
-The value for `site` must be one of the following:
-
-- The following URL based on your username: `https://<username>.github.io`
-- The random URL autogenerated for a [GitHub Organization's private page](https://docs.github.com/en/enterprise-cloud@latest/pages/getting-started-with-github-pages/changing-the-visibility-of-your-github-pages-site): `https://<random-string>.pages.github.io/`
-
-#### `base`
-
-A value for `base` may be required so that Astro will treat your repository name (e.g. `/my-repo`) as the root of your website.
-
-:::note
-  Don't set a `base` parameter if:
-
-- Your page is served from the root folder.
-- Your repository is located at `https://github.com/<USERNAME>/<USERNAME>.github.io`.
-:::
-
-The value for `base` should be your repository’s name starting with a forward slash, for example `/my-blog`. This is so that Astro understands your website's root is `/my-repo`, rather than the default `/`.
-
-:::caution
-    When this value is configured, all of your internal page links must be prefixed with your `base` value:
-
-```astro ins="/my-repo"
-<a href="/my-repo/about">About</a>
-```
-
-See more about [configuring a `base` value](/en/reference/configuration-reference/#base)
-:::
-
-### Using GitHub pages with a custom domain
-
-:::tip[Set up a custom domain]
-You can set up a custom domain by adding the following `./public/CNAME` file to your project: 
-
-```js title="public/CNAME"
-sub.mydomain.com
-```
-
-This will deploy your site at your custom domain instead of `user.github.io`. Don't forget to also [configure DNS for your domain provider](https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site#configuring-a-subdomain).   
-:::
-
-To configure Astro for using GitHub pages with a custom domain, set your domain as the value for `site`. Do not set a value for `base`:
-
-```js title="astro.config.mjs" ins={4}
-import { defineConfig } from 'astro/config'
-
-export default defineConfig({
-  site: 'https://example.com',
-})
-```
-
-## Configure a GitHub Action
+Follow the instructions below to use the GitHub Action to deploy your Astro site to GitHub Pages. This will create a website from your repository at a GitHub URL (e.g. `https://example.github.io`). Once deployed, you can optionally [configure a custom domain](#change-your-github-url-to-a-custom-domain) to deploy your GitHub Pages site at your preferred domain (e.g. `https://example.com`).
```

Comment:
```suggestion
Follow the instructions below to use the GitHub Action to deploy your Astro site to GitHub Pages. This will create a website from your repository at a GitHub URL (e.g. `https://<username>.github.io/<my-repo>`). Once deployed, you can optionally [configure a custom domain](#change-your-github-url-to-a-custom-domain) to deploy your GitHub Pages site at your preferred domain (e.g. `https://example.com`).
```